packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -44,71 +45,55 @@ module.exports = function getNewReleaseType( transformCommit, options = {} ) {
 					reject( new Error( 'Given repository is empty.' ) );
 				} else if ( err.message.match( new RegExp( `'${ options.tagName }\\.\\.HEAD': unknown` ) ) ) {
 					reject( new Error(
-						`Cannot find tag "${ options.tagName }" (the latest version from the changelog) ` +
-						'in given repository.'
+						`Cannot find tag "${ options.tagName }" (the latest version from the changelog) in given repository.`
 					) );
 				} else {
 					reject( err );
 				}
 			} )
 			.pipe( conventionalCommitsParser( parserOptions ) )
 			.pipe( concat( data => {
-				const commits = conventionalCommitsFilter( data );
-				const releaseType = getNewVersionType( commits );
-
-				return resolve( { releaseType } );
+				const commits = conventionalCommitsFilter( data )
+					.map( commit => transformCommit( commit, context ) )
+					.filter( commit => commit );
```

Comment:
`commit => !!commit` is more explicit IMO.

Reply:
It's enough if you know what `transformCommit` can return.